packages/roosterjs-editor-dom/lib/list/VList.ts

```diff
@@ -210,6 +218,25 @@ export default class VList {
         this.rootList = null;
     }
 
+    /**
+     * Sets the New List Start Property, that is going to be used to create a new List in the WriteBack function
+     * @param separator The HTML element that indicates when to split the VList
+     * @param startNumber The start number of the new List
+     */
+    split(separator: HTMLElement, startNumber: number) {
+        if (!this.rootList) {
+            throw new Error('rootList must not be null');
+        }
+
+        //Traverse the items of the VList, when the separator is found, set the New List Start Property
+        this.items.forEach(element => {
+            if (element.getNode() == separator) {
+                element.setNewListStart(startNumber);
+                return;
```

Comment:
this return statement won't work as expected because you are in a forEach loop.
To make it work, you need a regular for-loop